plugins/linux-capture/pipewire.c

```diff
@@ -567,6 +568,11 @@ static void on_param_changed_cb(void *user_data, uint32_t id,
 
 	spa_format_video_raw_parse(param, &obs_pw->format.info.raw);
 
+	// Announce support for DMA-BUF only when connected to a
+	// recent PipeWire server
```

Comment:
Nitpick: I think this comment is not necessary here, since this is properly documented in the commit message already